tools/documentation/content/get-started.md

```diff
@@ -100,7 +100,7 @@ eleventyNavigation:
     <img
       class="spectrum-CenteredImage"
       alt="Screenshot of the rendered CTA button in a browser window"
-      src="img/button-screen-shot.png"/>
+      src="../img/button-screen-shot.png"/>
```

Comment:
Can we revert this update too?

Reply:
Like we discussed with Patrick I have changed the img src in all yml and md files to reference from '../img' instead of 'img'. Please let me know if you have some other ideas too. Thanks

Reply:
Done